python/src/trezorlib/cardano.py

```diff
@@ -212,21 +257,58 @@ def _parse_tokens(tokens) -> List[messages.CardanoToken]:
 def _parse_address_parameters(
     address_parameters,
 ) -> messages.CardanoAddressParametersType:
-    if "path" not in address_parameters:
+    if "addressType" not in address_parameters:
         raise ValueError(INCOMPLETE_OUTPUT_ERROR_MESSAGE)
```

Comment:
Hmm, we also call `_parse_address_parameters` for reward addresses in catalyst, so the error should not be about OUTPUT only?

Reply:
https://github.com/vacuumlabs/trezor-firmware/pull/66/commits/ca9a89b2b753fbf0d744fb3bbec1335081ed6089#diff-aa355fcce844deffafbd696c1d414eb1c0198b0af2726710c7b8d8f7fda577cdR264